src/tests.builds

```diff
@@ -22,7 +22,7 @@
     <SerializeProjects>true</SerializeProjects>
   </PropertyGroup>
 
-  <Import Project="$(ToolsDir)CodeCoverage.targets" Condition="Exists('$(ToolsDir)CodeCoverage.targets')" />
+  <Import Project="$(ToolsDir)tests.targets" Condition="Exists('$(ToolsDir)tests.targets')" />
```

Comment:
OK then. We should probably remove the PerfTesting imports I suspect it is causing a build warning about being imported more then once.